slidez-core/src/main/resources/db/migration/V10__create_quiz_table.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE IF NOT EXISTS quiz
```

Comment:
I think that better use plural to table names.

Reply:
It is preferable and more common to use singular form